test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/DotnetCliUtil.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace NuGet.XPlat.FuncTest
 {
-    internal class DotnetCliUtil
+    public class DotnetCliUtil
```

Comment:
@alpaix  i tried to , but turns out netstandard1.3 (which nuget.test.utility targets) does not contain a definition for Assembly.Location which we use in this class. so i am going to leave it in here for now